mssw/My-List-Util/t/hosts-file.t

```diff
@@ -0,0 +1,28 @@
+package Blarg::Honk;
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::File;
+
+diag( "Testing Hosts file presence, $^X" );
+
+SKIP: {
+  if ( $^O !~ /windows/i ) {
+    skip "Not a Windows OS", 1;
+  }
+
+  file_exists_ok( 'C:\windows\system32\drivers\etc\hosts' );
+}
+
+SKIP: {
+  if ( $^O !~ /linux/i ) {
```

Comment:
The question just specified UNIX, so you could add support for other UNIX or UNIX-like OSes.